src/layers/attention.h

```diff
@@ -878,7 +914,8 @@ class Attention {
     }
 
     virtual const float *getMask(const float *attnMask, int bId, int hId, int srcLen, int tgtLen) {
-        return attnMask + bId * srcLen * tgtLen;
+        return attnMask;
```

Comment:
why different with origin?

Reply:
It is just for performance tuning. Forgot to rollback. Already add comments.